packages/design-tokens/src/docs/ColorSwatches/data/ColorSwatches.json

```
@@ -0,0 +1,368 @@
+[
+  {
+    "icon": "NeutralIcon",
+    "color": "NeutralText",
+    "background": "NeutralBackground",
+    "border": "NeutralBorder"
+  },
+  {
+    "icon": "NeutralIcon",
+    "color": "NeutralText",
+    "background": "NeutralBackgroundMedium",
+    "border": "NeutralBorder"
+  },
+  {
+    "icon": "NeutralIcon",
+    "color": "NeutralText",
+    "background": "NeutralBackgroundStrong",
+    "border": "NeutralBorder"
+  },
```

```suggestion
  {
    "icon": "NeutralIcon",
    "color": "NeutralText",
    "background": "NeutralBackgroundStrong",
    "border": "NeutralBorder"
  },
    {
    "color": "NeutralTextInverted",
    "background": "BrandingNavigation",
    "border": "NeutralBorderWeak"
  },
```
Not necessarily here of course